v-next/hardhat/src/internal/builtin-plugins/network-manager/hook-handlers/config.ts

```diff
@@ -92,3 +104,31 @@ export async function resolveUserConfig(
 function resolveGasConfig(value: GasUserConfig = "auto"): GasConfig {
   return value === "auto" ? value : BigInt(value);
 }
+
+function resolveAccounts(
+  accounts: HttpNetworkAccountsUserConfig | undefined,
+): HttpNetworkAccountsConfig {
+  const defaultHdAccountsConfigParams = {
```

Comment:
this is only used here, so I kept it in the function. Do we want to move it into a default value file?